src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -48,9 +52,12 @@ public void showWithdrawMessage() {
 
     /**
      * Creates a scanner in the Ui class
+     *
+     * @return
      */
-    public void createScanner() {
+    public Scanner createScanner() {
         this.scanner = new Scanner(System.in);
+        return null;
```

Comment:
Huh??? What's the rational for changing the return type to Scanner and then returning null all the time?